core/modules/system/system.menu.inc

```diff
@@ -120,6 +121,18 @@ function system_menu_block_form($config) {
     '#default_value' => $config['toggle'],
     '#description' => t('On devices with small screens, this option reduces the menu to a toggle button, commonly known as <a href="https://en.wikipedia.org/wiki/Hamburger_button" target="_blank"><em>hamburger button</em></a>. Clicking the icon reveals or hides the menu, saving screen space.'),
   );
+  
+  $form['clickdown'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Use a click to open instead of a hover'),
+    '#default_value' => $config['clickdown'],
+    '#description' => t('When checked, the menu will expand when clicked rather than when hovered.'),
```

Comment:
Instead of simply repeating the label of the checkbox, how about something like this?:

```suggestion
    '#description' => t('This option may work better on mobile, where there is no cursor, and touch events trigger menu expand/collapse.'),
```

Reply:
@klonos I wasn't sure how much of the debate about when this improves usability to try to pack into the description. (e.g. mobile use, older users with less steady hands, avoiding "hover tunnels"...) How about this? 

> Menus that don't appear or disappear on hover can be better for usability in some cases.